public/v1/apps/gitlab-ce.json

```diff
@@ -0,0 +1,43 @@
+{
+    "captainVersion": "1",
+    "documentation": "Taken from https://hub.docker.com/r/gitlab/gitlab-ce/",
+    "dockerCompose": {
+        "version": "3.3",
+        "services": {
+            "$$cap_appname": {
+                "image": "gitlab/gitlab-ce:$$cap_gitlab_version",
+                "volumes": [
+                    "$$cap_appname-config: /etc/gitlab",
+                    "$$cap_appname-logs: /var/log/gitlab",
+                    "$$cap_appname-data: /var/opt/gitlab"
+                ],
+                "restart": "always",
+                "containerHttpPort": "80",
+                "environment": {
+                    "GITLAB_OMNIBUS_CONFIG": "$$cap_gitlab_omnibus_config"
+                },
+                "ports": [
+                    "22:22"
```

Comment:
The host's SSH is also at port 22. Doing this will can cause the host's SSH to malfunction.